admin/spec/back-end/controllers/attendance.spec.js

```diff
@@ -113,6 +115,7 @@ describe('attendance controller:', () => {
       const req = getReq(reqParams)
       const validationError = new ValidationError()
       validationError.addError('firstName', true)
+      spyOn(checkWindowV2Service, 'getActiveCheckWindow').and.returnValue({ checkEndDate: moment.utc() })
```

Comment:
Setting the `checkEndDate` to now (moment.utc()) seems ambiguous to me.  Do you really want the `checkEndDate` to be in the past, future or actually now?  And if you do set it to 'now', then surely it is in the past by the time the system under test is executed?

Reply:
@jon-shipley value on checkEndDate for these tests is meaningless. Updated with a dummy value.